webv2/primarykey/primarykey_service.go

```diff
@@ -139,3 +139,59 @@ func isValidColumnOrder(pkRequest PrimaryKeyRequest) bool {
 
 	return true
 }
+
+// updateSpannerTableIndexKeyOrder Update Primary Key Order as columnId.
+func UpdateSpannerTableIndexKeyOrder(conv *internal.Conv) {
+
+	for _, spannerTable := range conv.SpSchema {
+		for i := 0; i < len(spannerTable.PrimaryKeys); i++ {
+			for spannercolumnid, _ := range spannerTable.ColDefs {
+				if spannerTable.PrimaryKeys[i].ColId == spannercolumnid {
+
+					o := getSpannerColumnIndex(spannerTable, spannercolumnid)
+					spannerTable.PrimaryKeys[i].Order = o
+
+				}
+			}
+		}
+	}
+}
+
+// updateSourceTableIndexKeyOrder Update Primary Key Order as columnId.
+func UpdateSourceTableIndexKeyOrder(conv *internal.Conv) {
+
+	for _, sourceTable := range conv.SrcSchema {
+
+		for i := 0; i < len(sourceTable.PrimaryKeys); i++ {
+			for sourcecolumnid, _ := range sourceTable.ColDefs {
+				if sourceTable.PrimaryKeys[i].ColId == sourcecolumnid {
+
+					o := getSourceColumnIndex(sourceTable, sourcecolumnid)
+					sourceTable.PrimaryKeys[i].Order = o
+				}
+			}
+		}
+	}
+}
+
+// getSpannerColumnIndex return columnn index as Inserted Order.
+func getSpannerColumnIndex(spannertable ddl.CreateTable, columnId string) int {
+
```

Comment:
duplicate methods

Reply:
Yes. And those redundant methods are removed. Please review it.